interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -426,12 +435,19 @@ namespace {
         }
       }
 #else // _MSC_VER
+      const char* RootInclude = std::getenv("ROOT_INCLUDE");
```

Comment:
It works properly indeed.
Here is the new version : https://github.com/Y--/root/commit/39736e1e362a0871d3a5a52b39305a67220ccd1a